styles.css

```diff
@@ -336,3 +338,157 @@ footer {
   line-height: 32px;
   color: #42526e;
 }
+
+/*
+Desktop hidden stuff
+*/
+#desktop-header {
+  display: none;
+}
+
+#icons-form-desktop {
+  display: none;
+}
+
+/*
+Desktop Media Query styles
+*/
+@media screen and (min-width: 768px) {
+  #first-container {
+    background-image: url('images/Header-llustration-desktop.png');
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .heading-section {
+    background-image: none;
+  }
+
+  #mobile-header {
+    display: none;
+  }
+
+  #desktop-header {
+    display: block;
+  }
+
+  #desktop-header .nav-list {
+    justify-content: center;
+  }
+
+  #desktop-header .nav-bar-a {
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    margin: 0 10px;
+  }
+
+  #main-heading {
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+    margin: 24px 0;
+  }
+
+  .heading-section p {
```

Comment:
- [ ]  [OPTIONAL] You're paragraph has more width than the provided design, please consider adding width to this ``` p``` tag so it can be like the Figma design

![Screenshot_20220429_173802](https://user-images.githubusercontent.com/25227022/165995111-56c8cf51-1540-408e-82af-d2d4162e11cf.png)